choose.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ *
+ */
+class Choose
+{
+  /***
+  レア度を抽選する
+  ***/
+
+  public static function choose_rare($rarity){
+    $sum_weight = 0;
+
+    for ($i=0; $i < count($rarity); $i++) {
+        $sum_weight += $rarity[$i][3];
+    }
+
+    $rand = rand(1, $sum_weight);
+
+    foreach($rarity as $rare){
+        if (($sum_weight -= $rare[3]) < $rand) return $rare[2];
```

Comment:
if ブロックは中括弧を省略しない書き方に統一すると一貫性があって良いと思います。

```suggestion
        if (($sum_weight -= $rare[3]) < $rand) {
            return $rare[2];
        }
```